src/utils/helpers.js

```diff
@@ -0,0 +1,5 @@
+export const titleCase = str =>
+  str
+    .split(' ')
```

Comment:
`.split` defaults to splitting on whitespace, this argument is unnecessary.

Reply:
see comment below